README.md

```diff
@@ -28,20 +29,20 @@ Here's the implementation file. This view's height can change based on the Dynam
 - (void)sharedInit {
     [super sharedInit];
     // Create this view's subviews
-    UIImageView *imageView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"MyImage.png"]];
+    _imageView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"MyImage.png"]];
     [self addSubview:imageView];
```

Comment:
Yeah I think it makes for a more realistic example for sure. Adds a few lines, but it's probably more in line with how people would actually use it. Maybe for consistency sake, we use the property everywhere (instead of the instance variable):

    self.imageView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"MyImage.png"]];